R-package/src/install.libs.R

```diff
@@ -142,8 +145,18 @@ if (!use_precompile) {
 dest <- file.path(R_PACKAGE_DIR, paste0("libs", R_ARCH), fsep = "/")
 dir.create(dest, recursive = TRUE, showWarnings = FALSE)
 if (file.exists(src)) {
-  cat("Found library file: ", src, " to move to ", dest, sep = "")
+  print(paste0("Found library file: ", src, " to move to ", dest))
   file.copy(src, dest, overwrite = TRUE)
 } else {
   stop(paste0("Cannot find lib_lightgbm", SHLIB_EXT))
 }
+
+# clean up the "build" directory
+if (dir.exists(build_dir)) {
+  print("Removing 'build/' directory")
+  unlink(
+    x = file.path(R_PACKAGE_SOURCE, "src", "build")
```

Comment:
I think it will be clearer: 

`x = file.path(R_PACKAGE_SOURCE, "src", "build")` -> `x = file.path(build_dir)`

(Suggestions are disabled after merging a PR.)

Reply:
blegh sorry, yeah you're right. That would eliminate a source of duplication. Let me add that right now.